plugin.video.dailymotion_com/addon.xml

```diff
@@ -10,11 +10,16 @@
     </extension>
     <extension point="xbmc.addon.metadata">
         <platform>all</platform>
-        <summary lang="en">Movies, TV, Music, User-Clips, 3D and Livestreams</summary>
+        <summary lang="en">Movies, TV, Music, User-Clips, and Livestreams</summary>
         <language></language>
-        <description lang="en">Movies, TV, Music, User-Clips, 3D and Livestreams</description>
+        <description lang="en">Dailymotion attracts 300 million users from around the world, who watch 3.5 billion videos on its player each month.</description>
         <disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
-        <website>http://www.dailymotion.com</website>
+		<summary lang="fr">Films, Séries TV, Musique, Clips, et Live</summary>
+		<language></language>
+		<description lang="fr">Dailymotion attire 300 millions d’utilisateurs qui regardent 3,5 milliards de vidéos sur son player chaque mois, à travers le monde.</description>
+		<disclaimer lang="fr">Certaines parties de cet addon peuvent ne pas être légales dans votre pays de résidence - s’il vous plaît vérifier avec vos lois locales avant d’installer.</disclaimer>
+        <news>Additional coding by alibaba011, gates, gujal, L0RE</news>
+		<website>http://www.dailymotion.com</website>
         <source>https://github.com/AddonScriptorDE/plugin.video.dailymotion_com</source>
```

Comment:
Might make sense to have your repo here now?

Reply:
What is the status of this PR, is it rejected?
Regarding your questions
1. Provider name. I left it as the original developer name and added mine under <news>. Please let me know if it is not acceptable
2. Fanart. Currently it is qHD size. Will try to find a bigger one, if not you can remove the fanart. The addon never had fanart to start with.

Reply:
@Gujal00 
Please send a new PR with changes to 1, 2 and 3.  It needs to have your nick, if your going to take over development :)
If anything is unclear, let us know.

Reply:
Hi thanks for that. So just a new PR with the above changes only (that is addon.xml and fanart.jpg) as all the other changes seem to have been merged